src/lib.rs

```diff
@@ -217,18 +217,20 @@ pub fn parse(text: &str, options: LiquidOptions) -> Result<Template> {
     parser::parse(&tokens, &options).map(Template::new)
 }
 
-/// Parse a liquid template from a file, returning Result<Template, io::Error>
+/// Parse a liquid template from a file, returning a Result (Template or io Error).
 /// # Examples
 ///
 /// ## Minimal Template
 ///
-/// template.liquid:
+/// `template.liquid`:
+///
 /// ```
```

Comment:
You need to add an `ignore` after the backticks here to avoid [this failure](https://travis-ci.org/cobalt-org/liquid-rust/jobs/152865914#L541)